app/models/katello/content_view_package_group_filter_rule.rb

```diff
@@ -7,6 +7,12 @@ class ContentViewPackageGroupFilterRule < Katello::Model
                :inverse_of => :package_group_rules,
                :foreign_key => :content_view_filter_id
 
+    has_one :package_group,
+            class_name: "Katello::PackageGroup",
+            foreign_key: :pulp_id,
+            primary_key: :uuid,
+            dependent: :nullify
```

Comment:
Hmm, I wonder if this should be destroy? I'm having trouble thinking through the implications for having this filter rule in place where the group no longer exists